test/kane/subscription_test.exs

```diff
@@ -76,10 +82,53 @@ defmodule Kane.SubscriptionTest do
                                         }))
     end)
 
-    assert {:ok, %Subscription{topic: %Topic{name: ^topic}, name: ^name, ack_deadline: 10}} =
+    assert {:ok, %Subscription{topic: %Topic{name: ^tname}, name: ^sname, ack_deadline: 10}} =
```

Comment:
Is there a reason this assertion was changed? This test is now failing, so I'm trying to understand the rationale for this part.